packages/wrangler/src/cloudchamber/common.ts

```diff
@@ -166,7 +166,7 @@ export async function promiseSpinner<T>(
 	return t;
 }
 
-async function fillOpenAPIConfiguration(config: Config, json: boolean) {
+export async function fillOpenAPIConfiguration(config: Config, json: boolean) {
```

Comment:
This function is doing some pretty funky things with auth scopes. I'm going to read through it in detail to see if it can be simplified, but what about just adding `cloudchamber:write` to Wrangler's default scopes?

Reply:
We have an engineer working on the containers auth scope that is going to remove the cloudchamber scope, I think we could leverage this as a follow-up?